zerver/openapi/markdown_extension.py

```diff
@@ -345,14 +367,16 @@ def generate_curl_example(
 def render_curl_example(
     function: str,
     api_url: str,
-    exclude: Optional[List[str]] = None,
-    include: Optional[List[str]] = None,
+    exclude: List[str] = list(),
+    include: List[str] = list(),
```

Comment:
Does this function ever get called with these parameters now?  If not, we should remove them from the signature.

Reply:
@timabbott  These do get called in cases where there are multiple configurations possible. So, I have kept the ones in the dictionaries as a base for the endpoints, and additional ones can be added by hardcoding in templates.

Reply:
For instance,
```
{generate_code_example(curl)|/users/me/subscriptions:post|example}

To subscribe another user to a stream, you may pass in
the `principals` parameter, like so:

{generate_code_example(curl, include=["principals"])|/users/me/subscriptions:post|example}
```